packages/fxa-auth-db-mysql/db-server/index.js

```diff
@@ -126,6 +126,10 @@ function createServer(db) {
   );
   api.post('/account/:id/locale', withIdAndBody(db.updateLocale));
   api.get('/account/:id/sessions', withIdAndBody(db.sessions));
+  api.put(
```

Comment:
I think we could add some remote tests here that go through the entire flow, create account, update anon id, etc.

https://github.com/mozilla/fxa/blob/cc595fc2b44bc1e15857c016539622a6a750bd5d/packages/fxa-auth-db-mysql/db-server/test/backend/remote.js#L144